Makefile.js

```diff
@@ -279,17 +279,16 @@ target.gensite = function() {
 
     echo("Generating eslint.org");
 
-    if (test("-d", DOCS_DIR)) {
-        rm("-r", DOCS_DIR);
```

Comment:
The other way is just to keep a list of the directories and files (like `['user-guide/rules/', 'user-guide/configuring']`) and delete them explicitly before copying over.